PR: https://github.com/odoo/odoo/pull/

From: aeda5d584f8fb531aa886533d7e8f13b69385c7c
From: Ravi Gadhia
Date: 2016-12-21 10:00:15

Structural Changes: 4
Total Changes: 4

[IMP] hr_recruitment: add support of activities

hr.applicant model now supports activities. Filters have been added to ease
their management.

================================= pseudo patch: =================================

--- a/addons/hr_recruitment/models/hr_recruitment.py
+++ b/addons/hr_recruitment/models/hr_recruitment.py
@@ -88,7 +88,7 @@ class Applicant(models.Model):
     _name = "hr.applicant"
     _description = "Applicant"
     _order = "priority desc, id desc"
-    _inherit = ['mail.thread', 'utm.mixin']
+    _inherit = ['mail.thread', 'mail.activity.mixin', 'utm.mixin']
     _mail_mass_mailing = _('Applicants')
 
     def _default_stage_id(self):
@@ -135,8 +135,6 @@ class Applicant(models.Model):
     date_closed = fields.Datetime("Closed", readonly=True, index=True)
     date_open = fields.Datetime("Assigned", readonly=True, index=True)
     date_last_stage_update = fields.Datetime("Last Stage Update", index=True, default=fields.Datetime.now)
-    date_action = fields.Date("Next Action Date")
-    title_action = fields.Char("Next Action", size=64)
     priority = fields.Selection(AVAILABLE_PRIORITIES, "Appreciation", default='0')
     job_id = fields.Many2one('hr.job', "Applied Job")
     salary_proposed_extra = fields.Char("Proposed Salary Extra", help="Salary Proposed by the Organisation, extra advantages")
